modules/aggregation-layers/src/contour-layer/marching-squares-codes.js

```diff
@@ -18,6 +18,14 @@ const OFFSET = {
   SW: [-HALF, -HALF]
 };
 
+// CORNERS of Cell
+const CORNERS = {
```

Comment:
Add comment similar to other enums defined above, something like :
`Table to map edge of intersection, pair of cell's who's weight is used as a domain for interpolation`.